src/Microsoft.AspNet.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -150,23 +159,25 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
                 await Options.Events.AuthenticationFailed(authenticationFailedContext);
                 if (authenticationFailedContext.HandledResponse)
                 {
-                    return authenticationFailedContext.AuthenticationTicket;
+                    return new AuthenticateResult()
+                    {
+                        Ticket = authenticationFailedContext.AuthenticationTicket
+                    };
                 }
 
                 if (authenticationFailedContext.Skipped)
                 {
-                    return null;
+                    return new AuthenticateResult();
                 }
 
                 throw;
```

Comment:
Make sure we don't forget about this.  Once the higher level changes are agreed on, the fun process of going through each of the specific error/exception paths needs to be reviewed :/